Gutenberg/version.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# This file isolates the definition of which version of Gutenberg to use.
+# This way, it can be accessed by multiple sources without duplication.
+
+# Either use commit or tag, if both are left uncommented, tag will take precedence.
+#
+# If you want to use a local version, please use the LOCAL_GUTENBERG environment variable when calling CocoaPods.
+#
+# Example:
+#
+#   LOCAL_GUTENBERG=../my-gutenberg-fork bundle exec pod install
+GUTENBERG_CONFIG = {
+  # commit: '',
+  tag: 'v1.94.0'
+}
+
+GITHUB_ORG = 'wordpress-mobile'
+REPO_NAME = 'gutenberg-mobile'
```

Comment:
These variables would be visible to all ruby code that "requires" this file, maybe it'd be good to put them into a namespace, or have a `GUTENBERG_` prefix?

Reply:
@crazytonyli Yeah, that makes sense. (Even though I'd say not many files not interested in these settings would `require` this file).

In https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/pull/114, @jhnstn suggested moving the settings into a JSON file. If we go down that path (or [the YAML alternative](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/pull/114#issuecomment-1543385142)) then we might work around this possiblity for confusiong.